Mathlib/RingTheory/DedekindDomain/Ideal.lean

```diff
@@ -839,8 +839,6 @@ and the lcm is their infimum, and use this to instantiate `NormalizedGCDMonoid (
 
 @[simp]
 theorem sup_mul_inf (I J : Ideal A) : (I ⊔ J) * (I ⊓ J) = I * J := by
-  letI := Classical.decEq (Ideal A)
-  letI := Classical.decEq (Associates (Ideal A))
```

Comment:
Why this change?

Reply:
Because these variables have now been added globally and are no longer needed.